### DISCUSSION TITLE
Have an option to hide the top bar

### DISCUSSION THREAD

**Discussion by cod-y** [will-stone/browserosaurus#463] [Ideas]
It would be nice to have an option to hide the icon from the top bar, much like [the finicky project](https://github.com/johnste/finicky/wiki/Configuration) does with `hideIcon: true`.

**Comment by will-stone** [comment]
Thanks. This isn’t something I’m looking to add. I currently use the menubar icon to notify about new updates (it turns blue). You could use an app like Bartender to hide Browserosaurus and other icons from the menubar. I personally go back and forth between using Bartender and not; sometimes I really like the minimalist look and other times I really like knowing which apps are running on my machine at a glance.

**Reply by ph00lt0** [reply]
Now that the update notification is also in the menu I think this could be made possible. Personally I update through brew anyhow so I really do not need this icon to be there. Would be great it can be disabled.

**Reply by will-stone** [reply]
Hi @ph00lt0 👋 If you’d like to submit a pull request that includes…
- a switch in the reducer to toggle the menu bar item. 
- a checkbox in the preference UI to communicate with this logic.
- a method to show/hide previous picker window (given the menubar item is no longer available). 
- a method to quit Browserosaurus (given the menubar item is no longer available). 
- tests for all of the above. 

…I’d be more than happy to review and potentially merge into the project. In the meantime there’s loads of great free/paid options out there that can hide this icon for you without needing to touch a line of code 🙂

**Comment by honsa** [comment]
Great idea this little app.

I like to run it in the background, just to open links. I dont see any reason to keep the icon wasting space in the menu bar.

Please add such an option, it should be not that hard. Jumpcut for example offers the user the option to hide the icon in the menu bar.

**Reply by will-stone** [reply]
Hi, thanks for using Browserosaurus. And thank you for your suggestion, but adding options adds maintenance complexity, and it's a solved issue:

- https://www.macbartender.com/
- https://matthewpalmer.net/vanilla/
- https://github.com/Mortennn/Dozer/

PS, I would steer clear from using phrases such as "it should be not that hard" on open source projects, unless you're willing to offer a pull request. In this instance there's no need though, as it's out-of-scope for this project.

Thanks 🙂 

Will.

**Comment by honsa** [comment]
Thank you for your fast reply and the links to 3. party tools which solve that.

May i find some time to make a PR ;) 

Edit: I saw the code, omg, but now I understand. It requires to change the settings concept, but anyway nice idea and little plugin, thanks for your work  :D
